src/components/Checkout/DonationNudge/DonationNudgeLightbox.vue

```diff
@@ -4,7 +4,7 @@
 		:no-padding-sides="true"
 		:no-padding-bottom="true"
 		:no-padding-top="true"
-		@lightbox-closed="closeNudgeLightbox"
+		@lightbox-closed="closeZeroUpsell(0)"
```

Comment:
We only want to do this when the zero donation upsell is open (i.e. the user has already selected $0), not all the time.